ocaml/forkexecd/lib/fe_systemctl.ml

```diff
@@ -26,12 +26,25 @@ let action ~service action =
   in
   ()
 
+(** Systemd execution type *)
+module Type = struct
+  type t = Simple | OneShot | Forking
+
+  let to_string = function
+    | Simple ->
+        "simple"
+    | OneShot ->
+        "oneshot"
+    | Forking ->
+        "forking"
```

Comment:
Can you add `notify` here? Full list of types: https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Type=, and in our version of systemd the manpage says:
`simple, forking, oneshot, dbus, notify, or idle`.

`dbus` we can probably leave out (we don't have dbus, we have message-switch).